maubot/management/frontend/src/pages/Main.js

```diff
@@ -39,6 +40,21 @@ class Main extends Component {
         this.setState({ pinged: true })
     }
 
+    async getBasePath() {
+        try {
+            const resp = await fetch(process.env.PUBLIC_URL + "/paths.json", {
```

Comment:
Please note the use of `process.env.PUBLIC_URL` -- I hope it's the right thing to do here (was previously `fetch("./paths.json")`).

`paths.json` is intended to be found at the same location as, e.g., `manifest.json` or `favicon.png`, which are linked as `%PUBLIC_URL%/...`, later expanded to `./...` by webpack.

Reply:
Yeah, looks fine